float64 joint0
float64 joint1
float64 joint2
float64 joint3
float64 joint4
float64 joint5
float64 hipX
float64 hipY
float64 hipZ
float64 hipRoll
float64 hipPitch
float64 hipYaw
float64 angularX
float64 angularY
float64 angularZ
float64 rightHeel
float64 rightToeX
float64 rightToeY
float64 rightToeZ
float64 leftHeel
float64 leftToeX
float64 leftToeY
float64 leftToeZ
float64 hipDistToFloor
float64 hipDistToObstacle
float64 leftProximityDistance
float64 rightProximityDistance
float64 rightFootX
float64 rightFootY
float64 rightFootZ
float64 leftFootX
float64 leftFootY
float64 leftFootZ
float64 rightFootDist
float64 leftFootDist
